app/services/jobs/return-logs/fetch-return-requirements.service.js

```diff
@@ -0,0 +1,149 @@
+'use strict'
+
+/**
+ * Fetches data needed for the generating return logs
+ * @module FetchReturnRequirementsService
+ */
+
+const ReturnLogModel = require('../../../models/return-log.model.js')
+const ReturnRequirementModel = require('../../../models/return-requirement.model.js')
+const ReturnVersionModel = require('../../../models/return-version.model.js')
+
+const { db } = require('../../../../db/db.js')
+const { cycleEndDate, cycleStartDate } = require('../../../lib/dates.lib.js')
+
+/**
+ * Fetch all return requirements that need return logs created.
+ *
+ * @param {boolean} summer - are we running summer cycle or all year
+ * @param {string} licenceReference - if provided only do the return log for that licence reference
+ *
+ * @returns {Promise<Array>} the list of return requirement ids
+ */
+async function go (summer, licenceReference) {
+  const returnRequirements = await _fetchReturnRequirements(summer, licenceReference)
+
+  return returnRequirements
```

Comment:
We can return directly here.

```suggestion
  return _fetchReturnRequirements(summer, licenceReference)
```